sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs

```diff
@@ -47,9 +43,9 @@ internal static EventDefinition GetEventDefintionFromPayload(string payload)
 
                 throw new Exception(string.Format(CultureInfo.CurrentCulture, AuthenticationEventResource.Ex_Comparable_Not_Found, comparable));
```

Comment:
This is a strange pattern.   Why throw one exception here just to re-wrap in a different exception in the `catch` block?

Reply:
yah, seems silly. I will change it to throw InvalidOperationException and use the same error message. 
Note @ShaneMicro